src/coreclr/nativeaot/Runtime/windows/PalRedhawkMinWin.cpp

```diff
@@ -671,6 +671,42 @@ REDHAWK_PALEXPORT char* PalCopyTCharAsChar(const TCHAR* toCopy)
     return converted;
 }
 
+REDHAWK_PALEXPORT HANDLE PalLoadLibrary(const char* moduleName)
+{
+    if (moduleName == nullptr)
+    {
+        return 0;
+    }
+    size_t len = strlen(moduleName);
+    wchar_t* moduleNameWide = new (nothrow)wchar_t[len + 1];
+    if (moduleNameWide == nullptr)
+    {
+        return 0;
+    }
+    for (size_t i = 0; i < len; i++)
+    {
+        moduleNameWide[i] = (wchar_t)moduleName[i];
+    }
+    moduleNameWide[len] = '\0';
+    
+    HANDLE result = LoadLibraryExW(moduleNameWide, NULL, 0);
```

Comment:
This needs to pass in specific flags. Otherwise, it is going to be flagged during security scans.